compiler/frontend/src/org/jetbrains/kotlin/resolve/calls/model/ResolvedCallImpl.java

```diff
@@ -65,7 +65,7 @@
     private D resultingDescriptor; // Probably substituted
     private final ReceiverValue dispatchReceiver; // receiver object of a method
     private ReceiverValue extensionReceiver; // receiver of an extension function
-    private List<ReceiverValue> additionalReceivers; // additional receivers of an extension function
+    private final List<ReceiverValue> additionalReceivers = Collections.emptyList(); // additional receivers of an extension function
```

Comment:
Why this field is still necessary?

Reply:
Yes, unnecessary. I just supposed this class is used in the resolution.